README.md

```diff
@@ -156,15 +156,14 @@ To build `fabric-tna.p4` using the Barefoot compiler and to create the pipeconf
 `.oar` package in one command:
 
 ```bash
-make build PROFILES=all
```

Comment:
`make build PROFILES=all` will become `make clean all pipeconf`
And the `all` target is:

```makefile
all: $(PROFILES)
```

Which will become

```makefile
all: all
```

Reply:
🤯